tutorials/dapt-curation/README.md

```diff
@@ -45,8 +45,10 @@ The tutorial follows the steps below:<br>
 
 After installing the NeMo Curator package, install the dependencies and run:
 
```

Comment:
In the steps above, please also clarify that there are now different versions of deduplication (it's currently only mentioning `exact`)